src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/lookups.mdx

```diff
@@ -0,0 +1,163 @@
+FROM lookup(storeNames) 
+SELECT *
+```
+
+This query will select some specific attributes from that same table: 
+
+```sql
+FROM lookup(storeNames) 
+SELECT store_ID, store_name, description
+```
+
+## Query examples [#query-with-data]
+
+The primary benefit of lookup tables is that you can use queries that combine that data with your New Relic-stored telemetry data. 
+
+Here are some query examples:
+
+<CollapserGroup>
+  <Collapser
+    id="avoid-hardcording"
+    title="Avoid hardcoding a long list of hosts"
+  >
+
+This query avoids hardcoding a long list of hosts by querying host names contained in a lookup table: 
+
+```sql
+FROM Log 
+SELECT count(*)
+WHERE hostname IN (FROM lookup(myHosts) SELECT uniques(myHost))
+```
+</Collapser>
+
+  <Collapser
+    id="using-join"
+    title="Query using JOIN"
+  >
+
+Using `JOIN` queries can make your data more understandable. For example, this query for a custom event type uses the `storeNames` table to show the store names along with the total sales. Also notice this allows a limit of 10,000: that's because lookup tables support a higher limit than other NRQL data types.
+
+```sql
+FROM StoreEvent 
+JOIN (FROM lookup(storeNames) SELECT store_ID as storeId, storeName AS name LIMIT 10000) ON shopId = storeId 
+SELECT shopId, storeName, totalSales
+```
+</Collapser>
+  <Collapser
+    id="status-codes"
+    title="Translate status codes"
+  >
+Here's a query that translates status codes to readable summaries of the status: 
+
+```sql
+FROM Transaction 
+  JOIN (FROM lookup(statusCodeTable) SELECT status_code, status_summary, status_definition)
+  ON http.statusCode=status_code
+SELECT count(*) FACET status_summary
+```
+
+Here are some example results: 
+
+<img
+  title="Screenshot of query for lookups translating status codes"
+  alt="Screenshot of query for lookups translating status codes"
+  src={nrqlExampleStatusCodes}
+/>
+</Collapser>
+
+  <Collapser
+    id="item-ids"
+    title="Translate item IDs"
+  >
+
+This query shows how several NRQL features can work together to get business information from queries of log data. The query extracts information about items from log messages, and uses `JOIN` to get user-friendly item names and to generate a table of item names and the number of items stored.
+
+```sql
+WITH aparse(message, 'POST to carts: * body: {"itemId":"*","unitPrice":*}%') AS (URL, ItemID, Price)
+FROM Log 
+  JOIN (FROM lookup(itemNames) SELECT ItemID, itemName) ON ItemID
+SELECT count(*) FACET itemName 
+WHERE message LIKE 'POST to carts%'
+SINCE 30 days ago
+```
+
+Here are some example results: 
+
+<img
+  title="Screenshot of query for lookups translating item IDs"
+  alt="Screenshot of query for lookups translating item IDs"
+  src={nrqlLookupQueryTranslateItemIds}
+/>
+
+</Collapser>
+
+  <Collapser
+    id="geoip"
+    title="Use geographic info to analyze locations"
+  >
+
+This query combines lookups and GeoIP to find which locations have the most unsuccessful statuses: 
```

Comment:
It would be nice to link "GeoIP" to the relevant page, once that's available.

Reply:
sorry, what is that page you speak of?

Reply:
The page explaining GeoIP.